src/bin/nitrogen.rs

```diff
@@ -40,6 +40,9 @@ enum Commands {
         /// EC2 instance type. Must be Nitro Enclaves compatible
         #[arg(long, default_value_t = String::from("m5a.xlarge"))]
         instance_type: String,
+        /// EC2 root disk size
```

Comment:
what's the units? would be good to add here

Reply:
same w/ the implicit max of `16384`

Reply:
Good point, added GiBs to the base description here and a constraint description to the template